src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,101 @@
+package duke.task;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.DukeStorageException;
+import duke.exceptions.DukeTaskCreationException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
A very small issue, but perhaps the junit libraries should be imported before the dependencies needed to run your tests (i.e. junit imports before your exception imports).